src/App.jsx

```diff
@@ -14,33 +44,69 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const visibleGoods = getPreparedGoods(goodsFromServer, {
+    sortField,
+    isReversed,
+  });
```

Comment:
Consider memoizing the `visibleGoods` calculation using `useMemo` to optimize performance. This will prevent unnecessary recalculations on every render when `sortField` or `isReversed` hasn't changed.